### DISCUSSION TITLE
close buffer with :q but then cursor has no focus until you click the screen

### DISCUSSION THREAD

**Discussion by KevinBaileyCrum** [JetBrains/ideavim#625] [General]
does this happen to anyone?

i close a buffer with :q, 
then on another tab/buffer if i type around, i get no response on the screen unless i click on the screen.
Idk if this is an intelij bug or an ideavim bug

anyone have any work arounds that they found to help out

**Comment by ljchen1** [comment]
This is a bug: https://youtrack.jetbrains.com/issue/VIM-2912/Focus-Lost-after-Closing-a-Tab-in-VIM-Mode-in-2023.1
Please watch there for updates.
